kong/cache/init.lua

```diff
@@ -140,8 +140,18 @@ function _M.new(opts)
     neg_ttl        = neg_ttl,
   }
 
+  local ok, err = cluster_events:subscribe("invalidations_" .. shm_name, function(key)
```

Comment:
This will already result in a new channel for each cache - `invalidations_kong_db_cache` and `invalidations_kong_core_db_cache`. Can we remove the old channel - it isn't part of a public interface so we should be fine removing it?

Reply:
I left the old channel deliberately because I've seen some of the custom plugins using this channel to do their validations, considering that this old channel has been here for a long time I think perhaps removing it suddenly will cause some noise

Reply:
In this case, I feel it would be better to only have one extra new channel, leaving the old one for the core cache, for example. Thoughts?

Reply:
Hmm, I think if any of these wild custom codes try to push events to the old channel, we won't know whether it is cache or core_cache entity they try to refresh. So maybe leaving the old channel for both is a better idea. How about I adding a deprecation warning when the old channel receives events? And we can totally deprecate the old channel in next major
What do you think?

Reply:
If there is significant risk that removing the event handler for the older `invalidations` channel will break existing custom plugins, then I think I could be in favor of keeping it.

Just to sanity check, though: doesn't this argument also apply to our removal of `self.cluster_events:broadcast("invalidations", ...)` in favor of the newer channel name? If some custom plugin has registered an event handler for this channel, they will break, after all. Or is this a much less likely scenario?